patches/net/minecraft/world/level/chunk/ChunkAccess.java.patch

```diff
@@ -12,7 +12,7 @@
 +        findBlocks(p_285343_, (state, pos) -> p_285343_.test(state), p_285030_);
 +    }
 +
-+    @Deprecated(forRemoval = true)
++    @Deprecated(forRemoval = true, since = "1.20.4")
```

Comment:
?

Reply:
It felt somewhat recent. I can yeet that too, though, if wanted.

Reply:
Yeet it!